### DISCUSSION TITLE
Dazn kodi compatible con PVR IPTV Simple Client

### DISCUSSION THREAD

**Discussion by tutanotaqatar22** [Paco8/plugin.video.orange.spain#15] [Ideas]
Hola, muchas gracias por tu trabajo, me gustaria si es posible hacer la aplicacon Dazn comtabile con la funcion PVR IPTV Simple Client como lo tienes como en Orange y Movistar. gracias

**Comment by Paco8** [comment]
Pues no sé, yo no uso ese addon.

**Reply by tutanotaqatar22** [reply]
Como te puedo ayudar para que se pueda hacer realidad? Gracias

**Reply by Paco8** [reply]
Deberías pedírselo a los desarrolladores de ese addon. Yo ni siquiera estoy abonado a Dazn.
